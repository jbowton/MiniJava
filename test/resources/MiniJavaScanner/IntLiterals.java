
//0
0
//single digit
6
//big number
748370203

// Double 00
00
// leading 0, positive number
05
