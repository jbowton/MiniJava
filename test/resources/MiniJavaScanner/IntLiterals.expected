[java] INTLITERAL(0) INTLITERAL(6) INTLITERAL(748370203) INTLITERAL(0) INTLITERAL(0) INTLITERAL(0) INTLITERAL(5)